src/test/resources/org/jabref/logic/importer/fetcher/collection_of_computer_science_bibliographies_single_result.bib

```diff
@@ -0,0 +1,6 @@
+@misc{,
+  author = {Tobias Olsson, Morgan Ericsson, Anna Wingkvist},
```

Comment:
Multiple authors should be represnted in BibTeX format.

    Tobias Olsson and Morgan Ericsson and Anna Wingkvist

We have an AuthorFormatter which can change the authors.

The other variant is to try to get the BibTeX directly from dblp.

If you stick with the dublin core parsing, please add an [ADR](https://adr.github.io/madr/) for it. In case you switch to the BibTeX, an ADR cannot harm.

Reply:
We have a dblp fetcher already that could be used here